sources/packages/backend/apps/workers/src/controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -45,7 +50,7 @@ export class ProgramInfoRequestController {
     try {
       const application = await this.applicationService.getApplicationById(
         job.variables.applicationId,
-        { loadDynamicData: false },
+        { loadDynamicData: true },
       );
       if (!application) {
```

Comment:
I wouldn't consider as part of PR. But this worker is not considering the application status to validate if an application operation is permitted. As we have the auto approvals, these things are essential now.

Reply:
I would refrain from adding status validation along the workers. We did it sometimes in the past, but my understanding is that it was a mistake.